commander/src/bootstrapping/commands/start.ts

```diff
@@ -33,7 +33,7 @@ import {
 
 const LOG_OPTIONS = ['trace', 'debug', 'info', 'warn', 'error', 'fatal'];
 export abstract class StartCommand extends Command {
-	static description = 'Start Lisk Core Node.';
+	static description = 'Start Blockchain Client.';
```

Comment:
Is it a client or it starts a node?

Reply:
I think its a blockchain client application similar to geth?